src/analysis/von_neumann_algebra/basic.lean

```diff
@@ -82,4 +82,7 @@ variables (H : Type u) [inner_product_space ℂ H] [complete_space H]
 instance : set_like (von_neumann_algebra H) (H →L[ℂ] H) :=
 ⟨von_neumann_algebra.carrier, λ p q h, by cases p; cases q; congr'⟩
 
+@[simp] lemma mem_carrier (M : von_neumann_algebra H)
+  (e : H →L[ℂ] H) : e ∈ M.carrier ↔ e ∈ M := iff.rfl
+
```

Comment:
I've made #18227 containing this lemma and some more interesting ones

Reply:
Cool cool! I was actually going to add a commutant definition, but you beat me to it!